bot-github-core/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Using a parent for dependency versions,
+       code formatting, and plugins. -->
+    <parent>
+        <groupId>org.commonhaus.automation</groupId>
+        <artifactId>bot-parent</artifactId>
+        <version>${revision}</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>bot-github-core</artifactId>
+
+    <name>Commonhaus Automation - GitHub Core</name>
+    <description>Query context + GraphQL assist</description>
+
+    <properties>
+        <!-- The following is set by mvnw in the parent (usually), but if you don't use mvnw... -->
+        <maven.multiModuleProjectDirectory>../</maven.multiModuleProjectDirectory>
```

Comment:
This property should be ser if there's a `.mvn` dir at the root of the reactor. Also, if the project has `mvnw` why not use it? (I know, I know) 😅